### PR TITLE
Update for hazelcast scripts

### PR DESCRIPTION
**Pull request by kobalski**
Updates server.sh script and writes hazelcast process id to a file then adds a stop script which reads current PID and shuts down hazelcast instance.

### CODE REVIEW

**Comment by hasancelik** [comment]
verify

File: hazelcast/src/main/resources/server.sh [hunk 1]
```diff
@@ -3,13 +3,14 @@
 PRG="$0"
 PRGDIR=`dirname "$PRG"`
 HAZELCAST_HOME=`cd "$PRGDIR/.." >/dev/null; pwd`
+PID_FILE=/tmp/hazelcast_instance.pid
```
Comment (mesutcelik) [inline hazelcast/src/main/resources/server.sh]: would be good to create it in local folder?

File: hazelcast/src/main/resources/stop.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+PRG="$0"
+PRGDIR=`dirname "$PRG"`
+HAZELCAST_HOME=`cd "$PRGDIR/.." >/dev/null; pwd`
+PID_FILE=/tmp/hazelcast_instance.pid
```
Comment (mesutcelik) [inline hazelcast/src/main/resources/stop.sh]: same here.

File: hazelcast/src/main/resources/stop.sh [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+PID=$(cat ${PID_FILE});
+
+if [[ -z "${PID}" ]]; then
+    echo "${PID_FILE}.pid is not running (missing PID)."
+else
+   kill -15 $1 ${PID}
```
Comment (mesutcelik) [inline hazelcast/src/main/resources/stop.sh]: no need to give `-15` as it is default signal for `kill`

**Comment by kobalski** [comment]
verify

**Comment by mesutcelik** [comment]
:+1:

**Comment by mesutcelik** [comment]
@kobalski , I think you can squash and send direct commit..

**Comment by kobalski** [comment]
@mesutcelik Changes squashed and sent with direct commit  https://github.com/hazelcast/hazelcast/commit/16cf5e275902449dc09ece9ec0e02ce93af7ef44

**Pull request closed without merging**